### PR TITLE
Fail if directory passed from cmd line doesn't exist

### PR DESCRIPTION
**Pull request by nitisht**
Mint accepts command line args to run specific tests. This PR adds a check to make sure the test directory exists before attempting to run tests.

### CODE REVIEW

File: mint.sh [hunk 1]
```diff
@@ -125,11 +125,16 @@ function main()
...
+            fi
+            (( i++ ))
+        else
+            echo Test directory "$sdk_name" not found. Exiting Mint.
+            exit 1
         fi
-        (( i++ ))
     done
 
     echo "Finished running all tests."
```
Comment (poornas) [inline mint.sh]: @nitisht , your changes are good - but noticed that we echo "Finished running all tests" even if one of the sdk runs failed. You could check whether your counter i matches the total # of sdks in the run_list and print accurate message of how many sdks passed.

Comment (nitisht) [reply mint.sh]: Updated the PR @poornas

File: mint.sh [hunk 2]
```diff
@@ -125,14 +125,24 @@ function main()
     i=1
     for sdk_dir in "${run_list[@]}"; do
         sdk_name=$(basename "$sdk_dir")
-        echo -n "($i/$count) Running $sdk_name tests ... "
-        if ! run_test "$sdk_dir"; then
-            break
+        if [ -d "$sdk_dir" ]; then
```
Comment (balamurugana) [inline mint.sh]: You could do
```bash
if [ ! -d "$sdk_dir" ]; then
 echo Test directory "$sdk_name" not found. Exiting Mint.
  exit 1
fi
```

File: mint.sh [hunk 3]
```diff
@@ -125,14 +125,24 @@ function main()
...
+            echo -n "($i/$count) Running $sdk_name tests ... " 
+            if ! run_test "$sdk_dir"; then
+                break
+            fi
+            (( i++ ))
+        else
+            echo Test directory "$sdk_name" not found. Exiting Mint.
```
Comment (balamurugana) [inline mint.sh]: Quote the entire string

File: mint.sh [hunk 4]
```diff
@@ -125,14 +125,24 @@ function main()
...
+            exit 1
         fi
-        (( i++ ))
     done
 
-    echo "Finished running all tests."
+    ## Report when all tests in run_list are run
+    if [ $i -eq $((count + 1)) ]; then
```
Comment (balamurugana) [inline mint.sh]: IMO you would need to do this as separate PR

Comment (nitisht) [reply mint.sh]: sure, will do @balamurugana

File: mint.sh (hunk 3 above)
Comment (nitisht) [reply mint.sh]: sure

**Comment by nitisht** [comment]
Updated PR @balamurugana

File: mint.sh [hunk 5]
```diff
@@ -125,6 +125,10 @@ function main()
     i=1
     for sdk_dir in "${run_list[@]}"; do
         sdk_name=$(basename "$sdk_dir")
+        if [ ! -d "$sdk_dir" ]; then
+            echo "Test directory $sdk_name not found. Exiting Mint."
+            exit 1
```
Comment (balamurugana) [inline mint.sh]: Test directory is not useful to the user.  You could print `test not found`
